### PR TITLE
feat!: store source metadata in concepts table

### LINKED ISSUE #215: merge concepts + metadata tables
Seems silly to have one table with 3 items in it. We should just move the metadata items to the concepts table

### PR DESCRIPTION
**Pull request by jsstevenson**
close #215 

this will break dynamodb/etc so maybe we hold off from merging, or merge to a prerelease version? (I haven't bumped version yet in this PR for this reason)

### CODE REVIEW

File: gene/database/dynamodb.py [hunk 1]
```diff
@@ -158,31 +147,15 @@ def _create_genes_table(self) -> None:
...
         """
         existing_tables = self.list_tables()
-        exists = (
-            self.gene_concepts_table in existing_tables
-            and self.gene_metadata_table in existing_tables
-        )
+        exists = self.gene_table in existing_tables
         if not exists:
-            logger.info("Gene tables are missing or unavailable.")
+            logger.info("Gene normalizer table is missing or unavailable.")
```
Comment (korikuzma) [inline gene/database/dynamodb.py:158]:
```suggestion
            logger.info(f"{self.gene_table} table is missing or unavailable.")
```

**Review by korikuzma — CHANGES_REQUESTED** [review]
Updating the aws ddbs won't be an issue, we can just manually insert the 4 metadata items. However, I do agree that we should wait until merging/releasing until the other 2 normalizers and metakb are updated with these changes.

**Comment by korikuzma** [comment]
If you really wanted to, you could just insert the metadata and keep the metadata tables until MetaKB is deployed with the updated versions

**Comment by jsstevenson** [comment]
all of this sounds a little scary but I'll just push to a prerelease version for now

**Review by korikuzma — APPROVED** [review]

**Pull request merged**